### PR TITLE
Fix/space list unreaded

### PR DESCRIPTION
**Pull request by TyaglovOV**

### CODE REVIEW

File: src/components/SpaceList/SpaceList.js [hunk 1]
```diff
@@ -3,68 +3,161 @@
  * @flow
  */
 
-import type { AvatarPlaceholder } from '@dlghq/dialog-types';
 import React, { PureComponent } from 'react';
 import classNames from 'classnames';
-
```
Comment (meafmira) [inline src/components/SpaceList/SpaceList.js]: Use blank line as separator between relative and absolute imports

File: src/components/SpaceList/SpaceList.js [hunk 2]
```diff
@@ -3,68 +3,161 @@
...
+      <Icon
+        className={plusClassName}
+        glyph={this.props.icon}
+        size={28}
+        {...handlers}
+      />
+    );
+  };
+
+  renderDropdown = () => {
```
Comment (meafmira) [inline src/components/SpaceList/SpaceList.js]: Move to main render method

Comment (TyaglovOV) [reply src/components/SpaceList/SpaceList.js]: This belongs not to main render method, but renderSpaceName (row 111 - 119);

Comment (meafmira) [reply src/components/SpaceList/SpaceList.js]: Ok. Move it there cause there is now any business logic

Comment (TyaglovOV) [reply src/components/SpaceList/SpaceList.js]: Oh, it's might be misunderstanding -- it belongs to Trigger element, and in docs you need to pass to Trigger render function, not react component itself.

File: src/components/SpaceList/SpaceAvatar/SpaceAvatar.js [hunk 3]
```diff
@@ -3,24 +3,22 @@
  * @flow
  */
 
-import type { AvatarPlaceholder } from '@dlghq/dialog-types';
 import React, { PureComponent } from 'react';
 import classNames from 'classnames';
+import type { Space } from '../types';
```
[the remark this answers is not in the document]
Comment (TyaglovOV) [reply src/components/SpaceList/SpaceAvatar/SpaceAvatar.js]: done

File: src/components/SpaceList/SpaceList.js (hunk 1 above)
Comment (TyaglovOV) [reply src/components/SpaceList/SpaceList.js]: done

**Comment by meafmira** [comment]
@TyaglovOV or @olegshilov rebase pls

**Comment by meafmira** [comment]
@TyaglovOV rebase

**Pull request closed without merging**